pingpong/src/main/java/com/cedrus/pingpong/kafka/PingPongProducer.java

```diff
@@ -17,18 +17,25 @@
 @Slf4j
 @Component
 public class PingPongProducer {
-    @Autowired private KafkaConfig config;
+    private KafkaConfig kafkaConfig;
+    private ObjectMapper objectMapper;
+    private Producer<String, String> producer;
+
+    @Autowired public PingPongProducer(KafkaConfig kafkaConfig, ObjectMapper objectMapper) {
+        this.kafkaConfig = kafkaConfig;
+        this.objectMapper = objectMapper;
 
-    public void sendMessage(PingPongMessage message) throws JsonProcessingException {
-        log.info("==== SENDING MESSAGE ====");
-        log.info("==== " + message.getTopic() + " ====");
```

Comment:
Do not use concatenation of strings in a logging expression.  It will incur an evaluation cost even if the line isn't printed.  Please use the proper format for variable arguments which allows for late evaluation.